src/client/app/components/groups/EditGroupModalComponent.tsx

```diff
@@ -937,6 +940,7 @@ export default function EditGroupModalComponent(props: EditGroupModalComponentPr
 	 */
```

Comment:
This comment applies to groupsToList() and following functions but I cannot put a comment there. It does ``listedMeters.sort();`` so seems to sort without being language aware. I know you didn't do this (looks like I'm at fault here) but I wonder why it should not be given a locale for sorting for display. What do you think? If it really should then I don't think it would be hard to change and I don't yet see a downside. Also, I searched for .sort in the code base and found potentially similar case (e.g., src/client/app/components/MultiCompareChartComponent.tsx). I have not checked how easy they are to address and the need to do it but it probably should be checked out.

Reply:
I pushed a commit containing the changed code for said function.